src/prefs/MenuEditorPage.js

```diff
@@ -963,22 +963,22 @@ var MenuEditorPage = class MenuEditorPage {
       _('You should try to have no more than twelve items in your menus.'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('You will find it more easy to learn item positions if you have an even number of entries. Four, six and eight are good numbers.'),
+      _('An even number of entries will make it easier to learn item positions. Four, six and eight are good numbers.'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('The source code of Fly-Pie is available on <a href="%s">Github</a>.')
+      _('Fly-Pie is libre software available on <a href="%s">GitHub</a>.')
           .replace('%s', 'https://github.com/Schneegans/Fly-Pie'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('Suggestions can be posted on <a href="%s">Github</a>.')
+      _('Suggestions can be posted on <a href="%s">GitHub</a>.')
           .replace('%s', 'https://github.com/Schneegans/Fly-Pie/issues'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('Bugs can be reported on <a href="%s">Github</a>.')
+      _('Bugs can be reported on <a href="%s">GitHub</a>.')
           .replace('%s', 'https://github.com/Schneegans/Fly-Pie/issues'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('Deep hierarchies are pretty efficient. Put menus into menus in menus!'),
+      _('Deep hierarchies are fairly inefficient. Avoid putting menus into menus in menus.'),
```

Comment:
This changed. Neither here nor there.

Reply:
Thank you for this pull request! It definitely improves the language.

However, I do not agree with this particular change. It _is_ a strength of marking menus that you can quickly select items at deeper levels. Consider a menu which has eight root items and then two nested levels of seven items each. Then you have 392 items in total. If you learned the gesture for some of them, you can select them in less than a second. Try this with a conventional linear menu.

Reply:
It is a tradeoff. Tried to do it the right way.